pkg/install/0-installstorage.go

```diff
@@ -42,36 +42,43 @@ func (i *Installer) createDNS(ctx context.Context) error {
 }
 
 func (i *Installer) installStorage(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
-	clusterID := &installconfig.ClusterID{}
 
-	err := clusterID.Generate(asset.Parents{
-		reflect.TypeOf(installConfig): &installconfig.InstallConfig{
-			Config: &types.InstallConfig{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: i.doc.OpenShiftCluster.Name,
+	g, err := i.loadGraph(ctx)
+	if hasAuthorizationFailedError(err) {
```

Comment:
This may be the only check needed - more testing is needed to see if any other errors should also trigger graph creation.

Reply:
This is fine, but I would like a small inline comment on why authorization error. Something like "If the graph already exists from the previous run, authorization error is returned". This way intention here is clear for the reader.

Reply:
> If the graph already exists from the previous run, authorization error is returned

It is actually the opposite -- authorization error occurs if it _doesn't_ exist already.

So I guess that just underlines why a comment is necessary :)